### DISCUSSION TITLE
Customising Entries listing view

### DISCUSSION THREAD

**Discussion by tidygraphic** [statamic/cms#5563] [Q&A]
Is there a way that I can customise the way entries are viewed when linking to them in the navigation or in entry fields? 

### Current issue

At the moment when choosing entries they are displayed like so: 

![Screenshot-2022-03-18-at-15 50 21](https://user-images.githubusercontent.com/5737710/159037908-7306b91e-afaf-4540-ad4d-eb668328ac23.png)

This isn't ideal as it's showing the title (`{{ banner_title }}`) from the banner region first and the actual page title (`{{ title }}`) second. This is making life a bit confusing for users and ideally the Page title would be over on the left (I don't even need to show the other title at all really). I can't see any way to customise this? 

### Temporary fix

I notice that if I move Page Title above Banner Title in the blueprint I get the following result: 

![Screenshot-2022-03-18-at-15 46 42](https://user-images.githubusercontent.com/5737710/159037504-935b126f-eea1-40db-8723-dfa87214c71c.png)

This is better than nothing but it means I have to have the title region in a place I don't really want it going forward.

### Question

It would be great if I could just customise how this screen displays - is that possible?

Thanks, Lea.

**Comment by duncanmcclean** [comment]
In the list where you choose the columns you wish to use, I believe you can reorder them. Does that help?

**Reply by tidygraphic** [reply]
Thanks! This is spot on and works a treat - I've changed this in the 'Pages' collection and everything appears correctly now (I was missing this because of the Tree view). 

I think this will only save that for my user account though - is that correct? If so can we change the default behaviour?

**Reply by tidygraphic** [reply]
This has also only fixed the display when choosing an entry for navigation - choosing entries for links/buttons etc still has the old behaviour.

**Reply by duncanmcclean** [reply]
Yes, it will only save for your user account unfortunately.

**Comment by wesort** [comment]
Similar to how collections list their entries and users can "customise columns", I'd welcome more control on what is displayed with the Entries fieldtype. Specifically when using the "Stack Selector Mode". 
– in the CP when choosing an entry 
– in the CP with what's displayed (when editing an entry)

**usecase**: Display the `title`, `featured_image` and `date`. The image in my case is the most recognisable aspect of the entries.
